packages/application-shell/src/components/project-container/project-container.js

```diff
@@ -112,9 +121,19 @@ export class ProjectContainer extends React.Component {
     if (this.state.hasError) {
       return <ErrorApologizer />;
     }
-    // TODO: do something if there is an `error`?
+
+    /**
+     * Given the user does not have project (and as a result is not part of an organization)
+     * the account application gets control over render. If any other application
+     * is requested to render a full page redirect (to have the proxy serve the request) occurs.
+     */
     if (this.props.user && this.props.user.projects.total === 0)
-      return <Redirect to="/logout?reason=no-projects" />;
```

Comment:
Q: what should we do here as long as signup is disabled?
Should we wrap this logic into feature toggle and clean up afterwards?

Reply:
Hm. Is there a problem with not wrapping it? In the AC you get your project and org straight away. If you get an account otherwise it may be nice to get to account creation (another toggle). Do you see a specific problem we can run into?

Reply:
Just thinking about the case when the user does not have any projects (it’s not necessarily after signup) and we redirect to create a project but the accounts app is non available (feature toggled).

Reply:
e46235f